sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_lro.py

```diff
@@ -0,0 +1,49 @@
+from urllib.parse import urlencode
```

Comment:
all new files need to have license headers on top, copy-paste this one

```
# coding=utf-8
# --------------------------------------------------------------------------
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.
# Code generated by Microsoft (R) AutoRest Code Generator.
# Changes may cause incorrect behavior and will be lost if the code is regenerated.
# --------------------------------------------------------------------------
```

Reply:
Updated all the new files.  I used the license header from the other hand-written files, but double checked that the autogenerated code files all had this one since the one you mention above explicitly mentions the autorest generator.